CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 **New**
 
+- Remove compiled CSS from distributed package
```

Comment:
is this project semver? So this will end up being a major version bump when we release, right?

Reply:
No-one has used this feature in reality so it could go out in a minor tbh, but yeah we could call this v6